src/ECS/registry/Registry.hpp

```diff
@@ -75,6 +76,7 @@ namespace ECS {
                 this->__componentManager.removeAllFromEntity(entity);
                 this->__entityManager.killEntity(entity);
                 for (auto const &cb : this->__killCallbacks) {
+                    std::cerr << "Calling kill callback for entity n°" << entity << std::endl;
```

Comment:
Debug left on purpose ?

Reply:
For now, its better to leave it! Later on we might want to remove all debugs to improve efficiency !

Reply:
okay